src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -91,12 +92,12 @@ class boss_nazan : public CreatureScript
                 _Reset();
             }
 
-            void JustEngagedWith(Unit* /*who*/) override { }
+            void JustEngagedWith(Unit* who) override { }
 
             void IsSummonedBy(WorldObject* summoner) override
             {
                 if (summoner->GetEntry() == NPC_VAZRUDEN_HERALD)
-                    VazrudenGUID = summoner->GetGUID();
+		Vazruden = me->FindNearestCreature(NPC_VAZRUDEN, 5000);
```

Comment:
we use 4 spaces instead of tabs. most IDEs support it in the settings.

Reply:
Updated